Add release info for sui-full-node doc (#19808)
MystenLabs/sui: docs/content/guides/operator/updates.mdx

description: Update your Sui Full node to the latest version to remain in sync with the network.
---

> ## Sui Release process

Reviewer: ```suggestion
## Sui release process

Each Sui branch is released on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through [official channels](#communication).  
```

Reviewer: You might have a better way to state this.

> `devnet` → every week on Mondays\

Reviewer: ```suggestion
- `devnet`: every week on Mondays.
```

> `testnet` → every week on Tuesdays\

Reviewer: ```suggestion
- `testnet`: every week on Tuesdays.
```

> `mainnet` → every two weeks on Wednesdays

Reviewer: ```suggestion
- `mainnet`: every two weeks on Wednesdays.
```

> Note: For more details, please see each network's release schedule and configuration [here](https://sui.io/networkinfo).

Reviewer: ```suggestion
:::info
For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
:::
```

Whenever Sui releases a new version, you must update your Full node with the release to ensure compatibility with the network it connects to. For example, if you use Sui Testnet you should install the version of Sui running on Sui Testnet.

Any release that contains a protocol change will need to be followed before the protocol upgrade takes place (when enough stake within the validator set upgrades, the new protocol version is enacted in the next epoch).
If you do not update your Full node, you will not be able to connect to the network after the protocol upgrade takes place.

## Communication

> Releases get announced on [Sui Discord server](https://discord.com/invite/sui) and [node-operators](https://groups.google.com/a/groups.sui.io/g/node-operators) Google group\

Reviewer: ```suggestion
Releases are announced on [Sui Discord server](https://discord.com/invite/sui) and [node-operators](https://groups.google.com/a/groups.sui.io/g/node-operators) Google group.
```

> Discord channels:\

Reviewer: ```suggestion
Discord channels:
```

> `devnet` → `#devnet-updates` channel\

Reviewer: ```suggestion
- `devnet`: [`#devnet-updates` channel](https://discord.com/channels/916379725201563759/1004638487078772736)
```

> `testnet` → `#tn-validator-announcements`, `#testnet-updates` ⁠and `#node-announcements` channels\

Reviewer: ```suggestion
- `testnet`: [`#tn-validator-announcements`](https://discord.com/channels/916379725201563759/1003660994381353101), [`#testnet-updates`](https://discord.com/channels/916379725201563759/1095151359642304612), ⁠and [`#node-announcements`](https://discord.com/channels/916379725201563759/1002231298888306718) channels.
```

> `mainnet` → `⁠#mn-validator-announcements`, `#mainnet-updates` and `#node-announcements` channels

Reviewer: ```suggestion
- `mainnet`: [`⁠#mn-validator-announcements`](https://discord.com/channels/916379725201563759/1093852827627040768), [`#mainnet-updates`](https://discord.com/channels/916379725201563759/1103082453792464906), and [`#node-announcements`](https://discord.com/channels/916379725201563759/1002231298888306718) channels.
```

description: Update your Sui Full node to the latest version to remain in sync with the network.
---

## Sui release process

> 

Reviewer: ```suggestion

Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through [official channels](#communication).

```

> - `devnet`: deployed every week on Mondays

Reviewer: ```suggestion
- `devnet`: Deployed every week on Mondays.
```

> - `testnet`: deployed every week on Tuesdays

Reviewer: ```suggestion
- `testnet`: Deployed every week on Tuesdays.
```

> - `mainnet`: deployed every two weeks on Wednesdays

Reviewer: ```suggestion
- `mainnet`: Deployed every two weeks on Wednesdays.
```

> > **_Info:_** For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).

Reviewer: ```suggestion
:::info
 For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
 :::
```

Whenever Sui releases a new version, you must update your Full node with the release to ensure compatibility with the network it connects to. For example, if you use Sui Testnet you should install the version of Sui running on Sui Testnet.

Any release that contains a protocol change will need to be followed before the protocol upgrade takes place (when enough stake within the validator set upgrades, the new protocol version is enacted in the next epoch).
If you do not update your Full node, you will not be able to connect to the network after the protocol upgrade takes place.

## Communication

> Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through.  

Reviewer: ```suggestion  
```

Releases are announced on [Sui Discord server](https://discord.com/invite/sui) and [node-operators](https://groups.google.com/a/groups.sui.io/g/node-operators) Google group.

> ##### Discord channels:

Reviewer: ```suggestion

### Discord channels
```

description: Update your Sui Full node to the latest version to remain in sync with the network.
---

## Sui release process

Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through [official channels](#communication).

- `devnet`: Deployed every week on Mondays.
- `testnet`: Deployed every week on Tuesdays.
- `mainnet`: Deployed every two weeks on Wednesdays.

:::info
 For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).

>  :::

Author: @ronny-mysten this does not render correctly
<img width="744" alt="Screenshot 2024-10-15 at 3 22 10 PM" src="https://github.com/user-attachments/assets/aba43bc8-b1b9-42ce-b1bc-85aa78c470fc">
